packages/suite-native-navigation-root/src/navigation/configs.ts

```diff
@@ -0,0 +1,6 @@
+export const stackNavigationOptionsConfig = {
```

Comment:
should not be in `navigation-root` but in `navigation`

Reply:
[8c23b9f](https://github.com/trezor/trezor-suite/pull/5617/commits/8c23b9f9cafd67684e4e1e0e3c5c399d53c6bc34)